modules/music_album_module.rb

```diff
@@ -16,17 +16,14 @@ def create_music_album
     on_spotify = on_spotify_input == 'true'
```

Comment:
on_spotify is always set true, but that is not what expected.

Hint:
Line 15: on_spotify = gets.chomp.downcase
Remove line 16

Reply:
fix on [f9c6a1b]